pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -53,23 +61,26 @@
 
     private static final String TENANT = "tnx";
     private static final String NAMESPACE1 = TENANT + "/ns1";
+    private static final int NUM_BROKER = 1;
+    private static final int NUM_TC_PER_ = 1;
 
     @BeforeMethod
     protected void setup() throws Exception {
-        this.setBrokerCount(1);
+        this.setBrokerCount(NUM_BROKER);
         this.internalSetup();
 
-        String[] brokerServiceUrlArr = getPulsarServiceList().get(0).getBrokerServiceUrl().split(":");
+        String[] brokerServiceUrlArr = getPulsarServiceList().get(NUM_BROKER - 1).getBrokerServiceUrl().split(":");
```

Comment:
why we are picking the last and not the first ?

Reply:
Excuse me.
I can't think of the difference between them,  I just use NUM_BROKER-1 uniformly.
But maybe, if there is no difference, it is a great philosophy to modify as little as possible.